htdocs/include/version.php

```diff
@@ -8,7 +8,7 @@
  * @since Xoops
  * @author phppp
  */
-define('ICMS_VERSION_NAME', 'ImpressCMS 2.0.0 RC2');
+define('ICMS_VERSION_NAME', 'ImpressCMS 2.0.0 Beta 2');
 
```

Comment:
RC, right?

Reply:
I was mistakenly convinced that it was RC already, but beta it is!